sanic/helpers.py

```diff
@@ -157,6 +159,10 @@ def import_string(module_name, package=None):
     return obj()
 
 
+def is_atty() -> bool:
+    return bool(sys.stdout and sys.stdout.isatty())
+
```

Comment:
Why are we moving this?

Reply:
I found that the best place to put this function inside helper.py, since placing it inside compact would cause a circular import issue.